x/liquidstaking/types/rebalancing.go

```diff
@@ -10,6 +10,7 @@ type Redelegation struct {
 	DstValidator LiquidValidator
 	Amount       sdk.Int
 	Last         bool
+	Error        error
```

Comment:
Is this for better debugging?

Reply:
Yes, it is not stored in the state but is used for error log and test debugging